### PR TITLE
[doc] : Updated documents.

### PR DESCRIPTION
**Pull request by ChetanT-System**
### Added
- Updated readme.md and open api yml file.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,7 +289,10 @@ To find information about supported submodules and there version in SDE please v
 | **GET:- localhost:8080/api/legal-entities** |This API is used to fetch legal entities (list of company's) for Process| Refer Api Doc |Refer Api Doc|
 | **POST:- localhost:8080/api/connectors-discovery** |This API is used to fetch connectors information | Refer Api Doc |Refer Api Doc|
 
+- Detailed API specs available under:
 
+https://github.com/catenax-ng/tx-managed-simple-data-exchanger-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml
```
Comment (dvasunin) [inline README.md:294]: wouldn't be better to use relative links here? This PR is targeting tractus-x repo, so link becomes external.

Comment (ChetanT-System) [reply README.md]: Corrected

File: README.md [hunk 2]
```diff
@@ -289,6 +289,11 @@ To find information about supported submodules and there version in SDE please v
 | **GET:- localhost:8080/api/legal-entities** |This API is used to fetch legal entities (list of company's) for Process| Refer Api Doc |Refer Api Doc|
 | **POST:- localhost:8080/api/connectors-discovery** |This API is used to fetch connectors information | Refer Api Doc |Refer Api Doc|
 
+## Detailed API specs available under:
+https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml
+
```
Comment (dvasunin) [inline README.md:294]: I would use relative link: [Detailed API spec](modules/sde-core/src/main/resources/sde-open-api.yml). It us universal and does not depend on repo

Comment (ChetanT-System) [reply README.md:295]: Done. Please check once.

**Review by dvasunin — APPROVED** [review]
I meant something like this: ``` [Detailed API specs available here](modules/sde-core/src/main/resources/sde-open-api.yml```, then a correct link is generated by MD processor, but it's ok. [here is examples](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)

**Comment by ChetanT-System** [comment]
> I meant something like this: ` [Detailed API specs available here](modules/sde-core/src/main/resources/sde-open-api.yml`, then a correct link is generated by MD processor, but it's ok. [here is examples](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)

@dvasunin  Updated readme.md file as you expected.

**Review by sachinargade123 — APPROVED** [review]
LGTM

**Comment by ChetanT-System** [comment]
@almadigabor  Please review and approve the PR.

**Review by almadigabor — APPROVED** [review]

**Pull request merged**